app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates. \
+It was originally an implementation of some algorithms published by N. Dershowitz and \
+E.M. Reingold in their book Calendrical Calculations."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="1989 - 2014 Ty Coon"
+LICENSE="GNU GPL v1"
+REVISION="1"
+SOURCE_URI="https://github.com/ewe2/mdate/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="6d58a8cf63c25b2b4b5b8904ba7c982f18608e8fa65436c555280699a020bcf8"
+SOURCE_DIR="mdate-$portVersion"
```

Comment:
You can (well, should, in fact) remove the `SOURCE_DIR="mdate-$portVersion"` because we prefer to not have that line when SOURCE_DIR is the default value.